clang/lib/Sema/TreeTransform.h

```diff
@@ -7083,8 +7087,9 @@ QualType TreeTransform<Derived>::TransformAttributedType(
       modifiedType != oldType->getModifiedType()) {
     // TODO: this is really lame; we should really be rebuilding the
     // equivalent type from first principles.
```

Comment:
This comment would no longer apply, would it?

Reply:
Indeed, it has been rebuilt using `TypeLoc` like `ModifiedType`.